src/index.re

```diff
@@ -193,18 +202,20 @@ let filterFruitsAndGetPoints = state =>
        ([], state.score)
      );
 
-let drawGrid = env =>
-  Draw.(
-    GridSizes.gridColumnsf
-    |> List.iter(gridPoint => {
-         let xStart = (gridPoint *. GridSizes.stepSizef, 0.);
-         let xEnd = (gridPoint *. GridSizes.stepSizef, GridSizes.gridHeightf);
-         linef(~p1=xStart, ~p2=xEnd, env);
-         let yStart = (0., gridPoint *. GridSizes.stepSizef);
-         let yEnd = (GridSizes.gridWidthf, gridPoint *. GridSizes.stepSizef);
-         linef(~p1=yStart, ~p2=yEnd, env);
-       })
-  );
+let drawGrid = env => {
+  open Draw;
+  fill(Utils.color(~r=0, ~g=0, ~b=0, ~a=100), env);
+  let (pointsX, pointsY) = GridSizes.gridPointsMatrix;
+  pointsX
+  |> List.iter(pointRow => {
+       let pointX = pointRow *. GridSizes.stepSizef;
+       pointsY
+       |> List.iter(pointColumn => {
+            let pointY = pointColumn *. GridSizes.stepSizef;
+            ellipsef(~center=(pointX, pointY), ~radx=1., ~rady=1., env);
+          });
+     });
```

Comment:
This is the main change. We iterate through the matrix and render each point as a small dot.